WalletWasabi/Models/ChaumianCoinJoin/CcjRound.cs

```diff
@@ -401,6 +401,7 @@ public async Task ExecuteNextPhaseAsync(CcjRoundPhase expectedPhase, Money feePe
 						// 7. Shuffle. NBitcoin's shuffle doesn't work: https://github.com/MetacoSA/NBitcoin/issues/713
 						transaction.Inputs.Shuffle();
 						transaction.Outputs.Shuffle();
+						transaction.Outputs.SortByAmount(); // So the coinjoin looks better in block explorer.
```

Comment:
It make no sense to shuffle the outputs and sort them immediately later.

Reply:
It does when the exact same amounts are registered.

Reply:
Oh wait, yes, it makes sense. Forget it.